src/components/CreateCourseRunPage/index.jsx

```diff
@@ -83,6 +86,16 @@ class CreateCourseRunPage extends React.Component {
       ({ label: choice.display_name, value: choice.value }));
   }
 
+  parseCourseRunLabels(courseRuns) {
+    // Sort course runs by descending start dates
+    const sortedCourseRuns =
+      courseRuns.sort((run1, run2) => moment(run2.start).diff(moment(run1.start)));
+    return sortedCourseRuns.map((run) => {
+      const runTerm = (run.key).split(/\+|\//).pop();
```

Comment:
Nit: you shouldn't need the wrapping parens on the run.key

Reply:
Good catch!